build.py

```diff
@@ -14,32 +14,35 @@
 
 FRAMEWORK_REPO = 'GeositeFramework'
 DEFAULT_ORG = 'CoastalResilienceNetwork'
-DEFAULT_BRANCH = 'master'
+DEFAULT_FRAMEWORK_BRANCH = 'develop'
+DEFAULT_REGION_BRANCH = 'development'
 BUILD_DIR = 'build'     # Build workspace
-OUTPUT_DIR = 'output'   # Installer artifact directory
+OUTPUT_DIR = 'output'   # Zip artifact directory
 
 
 def build_region(region, path, full_framework, framework_branch=None,
-                 region_branch=None, is_prod=False):
-    """ Build a GeositeFramework Region Installer and optionally install it """
+                 region_branch=None, is_prod=False, is_dev=False):
+    """ Build a GeositeFramework Region site and zip it up for installation """
     workspace = setup_workspace(path)
 
     # Get the repo name and to avoid confusion name it a _config dir
     region_dest = '%s_config' % region.split('/')[1]
     region_name = get_region_name(region)
 
-    # If installing to dev site, favor a branch named 'development' for the
+    # If --dev was specified, favor a branch named 'development' for the
     # region repo and any plugins, but don't fail if it doesn't exist
     # This is the convention used by TNC developers to introduce new features
-    if region_branch:
+    if region_branch and region_branch != DEFAULT_REGION_BRANCH:
         # An override was provided, use it
         region_branch = region_branch
     elif is_prod:
-        # Production build, use master (default)
-        region_branch = None
-    else:
+        # Production build, use master
+        region_branch = 'master'
```

Comment:
I'm not sure if this is desirable, but open to interpretation.  If a user builds with `--prod` and also `--region-branch=development`, `region_branch` will be set to `master`.

Either `--prod` and `--region-branch` should be mutually exclusive like `prod` and `dev` or it should respect the argument.

I don't think the same thing happens with `--dev` and `--region-branch` because of the first conditional in this block (`!= DEFAULT_REGION_BRANCH`), that is if you did `--dev --region-branch=master` you would end up with `master` as the region branch.  I think they should behave consistently.

Reply:
Okay. Reworked this a bit. It's just as (or more) verbose than before, but clear.